src/test/java/components/PanelHeaderTest.java

```diff
@@ -0,0 +1,117 @@
+
+    @BeforeMethod
+    public void beforeMethod() {
+        driver.get(HOMEPAGE_URL);
+    }
+
+    @Test
+    public void panelHeader_SignInLink_NavigatesToLoginPage() {
+        //when
+        LoginPage loginPage = panelHeader.openLoginPage();
+
+        //then
+        Assertions.assertThat(driver.getCurrentUrl()).contains(LOGIN_PAGE_URL);
+        Assertions.assertThat(loginPage.getPageTitle()).isEqualTo("Customer Login");
+    }
+
+    @Test
+    public void panelHeader_CreateAccountLink_NavigatesToLoginPage() {
+        //when
+        CreateAccountPage createAccountPage = panelHeader.openCreateAccountPage();
+
+        //then
+        Assertions.assertThat(driver.getCurrentUrl()).contains(CREATE_AN_ACCOUNT_URL);
+        Assertions.assertThat(createAccountPage.getPageTitle()).isEqualTo("Create New Customer Account");
+    }
+
+    @Test
+    public void panelHeaderThisIsDemoSite_HasTextAndColorsCorrectlyDisplayed() {
+        //then
+        Assertions.assertThat(panelHeader.isThisIsDemoSiteSectionDisplayed()).isTrue();
+        Assertions.assertThat(panelHeader.getThisIsDemoSiteSectionText()).isEqualTo("This is a demo store. No orders will be fulfilled.");
+        Assertions.assertThat(panelHeader.getThisIsDemoSiteSectionFontAsHex()).isEqualTo(WHITE_COLOR);
+        Assertions.assertThat(panelHeader.getThisIsDemoSiteSectionBackgroundAsHex()).isEqualTo(RED_COLOR);
+    }
+
+    @Test
+    public void panelHeader_PageWrapperGreyBar_HasColorsAndTextCorrectlyDisplayed() {
+        //then
+        Assertions.assertThat(panelHeader.isPanelHeaderSectionDisplayed()).isTrue();
+        Assertions.assertThat(panelHeader.getPanelHeaderSectionSectionFontAsHex()).isEqualTo(WHITE_COLOR);
+        Assertions.assertThat(panelHeader.getPanelHeaderSectionBackgroundAsHex()).isEqualTo(DARK_GREY_COLOR);
+        List<String> expectedNamesList = List.of("Sign In", "Create an Account");
+        List<String> expectedLinkList = List.of(LOGIN_PAGE_URL, CREATE_AN_ACCOUNT_URL);
+        validatePanelHeaderNamesAndLinks(headerPanelList, expectedNamesList, expectedLinkList);
+    }
+
+    @Test
+    public void panelHeader_LumaLogo_HasLogoAndLinksCorrectlyDisplayed() {
+        //then
+        Assertions.assertThat(panelHeader.isLumaLogoDisplayed()).isTrue();
+        Assertions.assertThat(panelHeader.getLumaLogoLink()).isEqualTo(HOMEPAGE_URL);
+        Assertions.assertThat(panelHeader.getLumaLogoImageSource()).isEqualTo
+                ("https://magento.softwaretestingboard.com/pub/static/version1666447838/frontend/Magento/luma/en_US/images/logo.svg");
+    }
+
+    @Test
+    public void panelHeader_SearchAndCart_HasTextAndLinksCorrectlyDisplayed() {
+        //then
+        Assertions.assertThat(panelHeader.isSearchFieldDisplayed()).isTrue();
+        Assertions.assertThat(panelHeader.getSearchFieldPlaceHolderText()).isEqualTo("Search entire store here...");
+        Assertions.assertThat(panelHeader.getSearchCartLink()).isEqualTo(SHOPPING_CART_URL);
```

Comment:
First it should be checked if minicart element is presented, then link should be checked.